subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -271,6 +271,29 @@ static uint8_t adv_sync_pdu_ad_data_set(struct pdu_adv *pdu,
 	return 0;
 }
 
+static uint8_t adv_sync_pdu_cte_info_set(struct pdu_adv *pdu,
```

Comment:
Please add comment about assumption related with a PDU content - that it is already properly prepared and does not include any field that may follow CTEInfo (AuxPtr, TxPower, ACAD). 

Also I suggest to add assertions to check if AuxPtr, TxPower, ACAD were available, it may save us trouble searching for error if those field were available and overwritten by accident.

Reply:
This only updates CTEInfo if present, it does not affect other fields in any way so it does not matter if they are there or not.

Reply:
Ok function is named `xxx_set` so the PDU should have correctly allocated memory CTEInfo and PDU is not damaged then. 
Nevertheless please provide some information about that in comment.